app/javascript/controllers/projects_controller.js

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable class-methods-use-this */
+import { Controller } from 'stimulus';
+
+var flag = false;
+
+export default class extends Controller {
+    handleMainCheckbox() {
+        document.querySelector('.advance-embed-option').style.display = (flag) ? 'none' : 'block';
+        document.querySelector('.advance-embed-field .dropdown').style.transform = (flag) ? 'rotate(0deg)' : 'rotate(180deg)';
+        flag = !flag;
```

Comment:
if you are using this for toggler, you can simplify by using https://developer.mozilla.org/en-US/docs/Web/HTML/Element/details